src/app/pages/start-wizard/start-wizard.page.ts

```diff
@@ -126,7 +126,8 @@ export class StartWizardPage implements OnInit, OnDestroy {
       const userSettings = await this.userSettingService.userSetting$.pipe(take(1)).toPromise();
       this.userSettingService.saveUserSettings({
         ...userSettings,
-        completedStartWizard: true
+        completedStartWizard: true,
+        showSimpleObsOnboarding: true
```

Comment:
Jeg tror koden kanskje hadde blitt litt enklere hvis du snudde flagget "showSimpleObsOnboarding" "på hodet". Altså satt flagget til true når bruker hadde sett ferdig onboardinga for enkel observasjon. Da kunne "default" vært false eller undefined, og så hadde du sluppet å sette flagget i start-wizard.page.ts